core/src/services/routing.js

```diff
@@ -395,6 +390,14 @@ class RoutingClass {
         } else if (cnode.tabNav === false) {
           tabNavInherited = false;
           break;
+        } else if (GenericHelpers.isObject(cnode.tabNav)) {
+          if (cnode.tabNav.hideTabNavAutomatically === true && cnode.children.length === 1) {
+            tabNavInherited = false;
+            break;
+          } else {
```

Comment:
I think this **else** part needs some re-work:
For example with the following **tabNav** definition **tabNav** is enabled too, just because it is an **Object**: 
EXAMPLE: 

 ` tabNav: { blabla: false },` this is equal to  `tabNav: true` which I suppose we shouldn't allow
 Also `tabNav: {}` also enables **tabNav** which I suppose we should also not allow 🤔 ?

Reply:
Yes, indeed. To optimistic ;).